pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1537,6 +1537,10 @@ type ObjectStoreUserSpec struct {
 	// +optional
 	// +nullable
 	Quotas *ObjectUserQuotaSpec `json:"quotas,omitempty"`
+	// +optional
```

Comment:
```suggestion
	// +optional
	// +nullable
```
I think nullable is also required?

Reply:
if it's not a point, need nullable to? @subhamkrai

Reply:
In the past, in one case I saw the issue because of missing nullable that's why I thought should we apply it here too. Let me find that issue and I'll  update here